### PR TITLE
feat: Sets up portal public views with rails ERB and tailwind

### PR DESCRIPTION
**Pull request by nithindavid**
This PR sets up basic configurations to render help center home, articles.
This also includes 
1. rails turbo links
2. tailwind css

**How to test**
1. Navigate to portals home page and make sure you have an active portal. If not create one
2. Goto `http://localhost:3000/public/api/v1/portals/portal-slug`

image
image

### CODE REVIEW

File: app/assets/stylesheets/portal/application.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@import "tailwindcss/base";
```
Comment (pranavrajs) [inline app/assets/stylesheets/portal/application.scss:1]: We are not using Rails Asset Pipeline, let us move this packs and import the pack in the layout file.

Comment (pranavrajs) [reply app/assets/stylesheets/portal/application.scss:1]: You have `app/javascript/portal/application.scss` already, so let us remove this.

**Comment by muhsin-k** [comment]
@nithindavid The draft articles are also showing in public view https://www.loom.com/share/0ad279ba76e543e9a2db98dd816d025e

**Review by tejaswinichile — APPROVED** [review]

**Review by fayazara — CHANGES_REQUESTED** [review]
Have left a minor change comment. 

The rest of frontend looks good!

**Review by tejaswinichile — APPROVED** [review]

File: app/views/public/api/v1/portals/articles/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+				<a class="text-slate-800 hover:underline leading-8"
+        href="/hc/<%= @portal.slug %>/<%= @category.slug %>" class=""><%= @portal.name %> Home</a>
+		<span>/</span>
+		<span>/</span>
+		</div>
+			<% @articles.each do |article| %>
+				<h1 class="text-4xl font-bold md:tracking-normal leading-snug md:text-5xl text-slate-900">
+				<%= article.title %></h1>
+				<div class="flex flex-col items-start justify-between w-full md:flex-row md:items-center pt-2">
+					<div class="flex items-center md:space-x-2">
```
Comment (iamsivin) [inline app/views/public/api/v1/portals/articles/index.html.erb:15]:
```suggestion
					<div class="flex items-center space-x-2">
```

**Review by sojan-official — APPROVED** [review]

**Pull request merged**